### PR TITLE
Upgrade bunsen

### PR DESCRIPTION
**Pull request by sandersky**
#MAJOR#

# CHANGELOG

## Breaking

* **Upgraded** [ember-frost-bunsen](https://github.com/ciena-frost/ember-frost-bunsen) to version `6.0.3`.
* **Upgraded** [ember-prop-types](https://github.com/ciena-blueplanet/ember-prop-types) to version `2.0.0`.

## Non-Breaking

* **Added** `renderers` property for passing renderers to `frost-bunsen-form`.
* **Added** `showAllErrors` property for whether or not form should show all errors.
* **Added** `validators` property for passing validators to `frost-bunsen-form`.
* **Updated** component to use [ember-prop-types](https://github.com/ciena-blueplanet/ember-prop-types) for property validation.

### CODE REVIEW

File: addon/components/frost-modal-input.js [hunk 1]
```diff
@@ -11,6 +11,8 @@ export default Component.extend({
   $containerEl: null,
   $headerEl: null,
   $footerEl: null,
+  renderers: {},
+  validators: [],
```
Comment (job13er) [inline addon/components/frost-modal-input.js]: I thought it was generally a bad idea to initialize anything but simple types like this. I realize we're not doing anything like `renderers.push()`, and it's always re-assigned, so it probably won't bite us, but I thought it was worth mentioning.

Comment (job13er) [inline addon/components/frost-modal-input.js]: For completeness, should `showAllErrors` be initialized here as well to `false`?

File: addon/templates/components/frost-modal-input.hbs [hunk 2]
```diff
@@ -33,11 +33,15 @@
     minScrollbarLength=20
     maxScrollbarLength=150}}
     {{frost-bunsen-form
-      model=formModel
-      view=formView
-      value=formValue
+      bunsenModel=formModel
+      bunsenView=formView
+      inline=true
```
Comment (job13er) [inline addon/templates/components/frost-modal-input.hbs]: Do we want to allow the consumer to override this as well? I.e. make it come from the properties of the `frost-modal-input` and just default it to `true` (similar to how you handled `showAllErrors`?)

Comment (sandersky) [inline addon/templates/components/frost-modal-input.hbs]: I actually think we should probably just get rid of the inline property from bunsen as I don't know if we actually use it both ways and it makes the styles more complex.

File: addon/components/frost-modal-input.js [hunk 3]
```diff
@@ -1,17 +1,66 @@
...
+      PropTypes.object
+    ]),
+    showAllErrors: PropTypes.bool,
+    subtitle: PropTypes.string,
+    title: PropTypes.string,
+    titleComponent: PropTypes.string,
+    validators: PropTypes.array
+  },
+
+  getDefaultProps () {
```
Comment (job13er) [inline addon/components/frost-modal-input.js]: :+1:

**Comment by job13er** [comment]
:+1:

**Pull request merged**